test/integration/src/test/java/io/pravega/test/integration/ControllerStreamMetadataTest.java

```diff
@@ -100,7 +100,7 @@ public void tearDown() {
         }
     }
 
-    @Test(timeout = 60000)
+    @Test(timeout = 10000)
```

Comment:
The test finishes in a split second. Even 10 seconds is too high a timeout value for this test. I changed it to 10 seconds because that is the standard timeout we have in all our unit tests.